src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -411,7 +411,7 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
               <span>
                 <b>Balance:</b>
                 <i title={balance && `${formatMax(balance, balance.currency.decimals)} ${balance.currency.symbol}`}>
```

Comment:
why the title and the value displayed use different var? 
TITLE => balance.currency.decimals
VALUE => currencyAmount?.currency?.symbol

Reply:
Ahhh good catch! Also need to update the title!